### PR TITLE
Update help card content

### PR DESCRIPTION
**Pull request by HSalmon**
New Help card content

### CODE REVIEW

**Comment by HSalmon** [comment]
screen shot 2015-09-22 at 3 55 56 pm

**Comment by robguthrie** [comment]
Looking at this.. it's not obvious what the video is. I wonder if we should
have a fake video place holder with a clear title in it, that opens a modal
with a bigger video

**Comment by HSalmon** [comment]
What does this need to get merged @rdbartlett, @robguthrie?

**Comment by robguthrie** [comment]
I'm happy for you to make the call on how much you want to do before deploying. I see it as a cool experiment and am not too fussed on details that delay it's deploy.

**Comment by HSalmon** [comment]
screen shot 2015-09-25 at 1 48 44 pm

File: lineman/app/components/group_page/group_help_card/group_help_card.haml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+%section.group-help-card
```
Comment (gdpelican) [inline lineman/app/components/group_page/group_help_card/group_help_card.haml]: Hmm... I wonder if we should display this only on loomio.org, and leave it off of future whitelabel versions?

Comment (HSalmon) [inline lineman/app/components/group_page/group_help_card/group_help_card.haml]: I'm not sure...@rdbartlett, @robguthrie, thoughts?

Comment (robguthrie) [inline lineman/app/components/group_page/group_help_card/group_help_card.haml]: No I think we just leave it for everyone. We want to see traffic from other installs hitting our docs.

**Pull request merged**